src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -37,6 +39,9 @@ function OptionRowLHN({reportID, isFocused = false, onSelectRow = () => {}, opti
     const isFocusedRef = useRef(true);
     const {shouldUseNarrowLayout} = useResponsiveLayout();
 
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Why do I see this being disabled for some components, but not others like `SettlementButton.tsk` for example?

Reply:
The reason is for some components like `SettlementButton` `chatReportID` has type `string` and then no lint error when we use `||`.